tests/suites/test_suite_psa_crypto_se_driver_hal.data

```diff
@@ -0,0 +1,25 @@
+Register SE driver: good
+register_one:2:PSA_DRV_SE_HAL_VERSION:PSA_SUCCESS
+
+# Run this test case a second time to verify that the library deinit
+# unregistered the first driver.
+Register SE driver: good, again
+register_one:2:PSA_DRV_SE_HAL_VERSION:PSA_SUCCESS
+
+Register SE driver: invalid lifetime (VOLATILE)
+register_one:PSA_KEY_LIFETIME_VOLATILE:PSA_DRV_SE_HAL_VERSION:PSA_ERROR_INVALID_ARGUMENT
+
```

Comment:
I would add a test checking if registering with lifetime "0" works.

Reply:
I think that it probably would not work because `PSA_KEY_LIFETIME_VOLATILE == 0`, but I agree that this is something to look into. As I mentioned earlier, I think that having a value (e.g. 0) mean two things here may not be ideal (and a bit confusing)

Reply:
`0` is invalid. So is `PSA_KEY_LIFETIME_VOLATILE`. A test case is cheap, so I added one with `0` as well.